### ISSUE TITLE
Murdered: Soul Suspect (233290)

### ISSUE THREAD

**Issue by mmolch** [ValveSoftware/Proton#321]
Runs without any issues for me with Proton 3.7

Tested on:
Ubuntu 18.04
Core i7-4790K, 16 GB Ram
Geforce GTX 1060 (driver version 396.51)

**Comment by hrkrx** [comment]
I do not think it is ready yet, when i try to enter the cemetery it always crashes to a point where it hogs all system recources. 

EDIT: that is about 2h into the game, everything else so far is perfect

**Comment by kisak-valve** [comment]
Hello @hrkrx, please copy your system information from steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. If it is crashing, why would it be using resources? If possible, please make a save just before the problem area, then add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the issue in a short run of the game, and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by hrkrx** [comment]
https://gist.github.com/hrkrx/6fe245f06c3114eddfd43cadeab46d57

A little more detail:

I can walk to the gate where the game asks me to confirm to change the level.
When i accept a cutscene begins to play and freezes after about 2 seconds.
It freezes always at exactly the same frame.
After that the process demands 100% of one cpu core and the whole system begins to lag to a point where it's almost unusable.
I then use another machine to SIGKILL via ssh the process.
Nothing else i tried can stop the process.
If i play in fullscreen its even worse.

Since i can ssh into the system i think there maybe some GPU Issues.

EDIT: It happens on all available versions of proton

**Comment by psyllocyber** [comment]
I had exactly the same issue as @hrkrx . This is how I fixed it. Add `-DX9` to the launch options in Steam. There seems to be an issue with the DX11 compatabilty layer.

**Comment by CSahajdacny** [comment]
The game freeze in cutscene "entering cementery" is present on PROTON 4.2-1 too...
Maybe is a bug in DXVK...

**Comment by minisori** [comment]
It's not the cutscene itself, it also happens when you load a game from there using dx11.
Aside of that bug i was able to complete the entire game, sadly i have no reports from that part.
[Full system report](https://gist.github.com/minisori/217be39a24ae8a58e7d70a022df8c481)

**Comment by CSahajdacny** [comment]
The game freeze in cutscene "entering cementery" is present on PROTON 4.2-3
[steam-233290.log](https://github.com/ValveSoftware/Proton/files/3101245/steam-233290.log)

**Comment by CSahajdacny** [comment]
The game freeze in cutscene "entering cementery" is present on PROTON 4.2-4
[steam-233290.log](https://github.com/ValveSoftware/Proton/files/3180155/steam-233290.log)
![Captura de pantalla de 2019-05-14 19-34-28](https://user-images.githubusercontent.com/45825041/57736954-452d5100-7680-11e9-8f82-237cfb359e9a.jpg)

**Comment by CSahajdacny** [comment]
The game crash when "entering cemetery" via PROTON 4.2-5 too.
[steam-233290.log](https://github.com/ValveSoftware/Proton/files/3215025/steam-233290.log)
![FallaMurderedSoulSuspectPROTON425](https://user-images.githubusercontent.com/45825041/58301598-78c55500-7dbd-11e9-9abc-fc278ee2d200.png)

**Comment by hrkrx** [comment]
i was able to achieve a full playthrough in "-dx9" mode as launch parameter

**Comment by CSahajdacny** [comment]
The game still crash when "entering cemetery" via PROTON 4.2-6 too. ¬¬
[steam-233290.log](https://github.com/ValveSoftware/Proton/files/3264421/steam-233290.log)

**Comment by CSahajdacny** [comment]
This is weird!!!
The game still crash when "entering cemetery" via PROTON 4.11-7 too, but only when I use my GTX 970
[steam-233290-GTX970.log](https://github.com/ValveSoftware/Proton/files/3822479/steam-233290-GTX970.log)
 
...When I use my RTX 2060 SUPER, the game doesn't crash anymore when "entering cemetery" via PROTON 4.11-7 (even via PROTON 4.2-6, the game doesn't crash anymore) 
[steam-233290-RTX2060SUPER.log](https://github.com/ValveSoftware/Proton/files/3822486/steam-233290-RTX2060SUPER.log)

**Comment by Kreyren** [comment]
This compatibility report is work in progress, trying to find a reliable way to reproduce the issue

---

# Compatibility Report
- **Name of the game with compatibility issues:** MURDERED: Soul Suspect
- **Steam AppID of the game:** 233290

## System Information
- **OS:** Kreyrock Linux 9999 Anubis x86_64
- **GPU:** AMD ATI Radeon HD 7870 GHz Edition
- **Driver/LLVM version:** PITCAIRN, DRM 3.33.0, LLVM 9.0.0
- **Kernel version:** 5.3.0-3-amd64
- **Link to full system information report as [Gist](https://gist.github.com/):** 
  - [Hardware info](https://gist.githubusercontent.com/Kreyren/89b8671140490850b20fa6073cc6b32f/raw/98e480f637ffaa764b74dab4d2891855f73a4d7e/gistfile1.txt)
  - [Installed packages debian](https://gist.githubusercontent.com/Kreyren/3a2f250b9831e4e3270d357ce0665ca8/raw/f310a3629e5582ee7dcc3cedcfb4ca09ac3fa7b9/gistfile1.txt)
  - [Installed packages gentoo](https://gist.githubusercontent.com/Kreyren/2f4d80f50d89e23936b93f97b934213f/raw/2ee31e72553ad6264db0df26221cdc83f04687cf/gistfile1.txt)
  - [Installed packages Kreyherbo](https://gist.githubusercontent.com/Kreyren/cce4365775a166eda114c1020142e08f/raw/b042b94f1f637b8e32787ea1536911e987948096/gistfile1.txt)
  - [steam system info](https://gist.githubusercontent.com/Kreyren/811358083cf6e01d98d65bb04b622c50/raw/6879cc1526c7ced58bc8f4041a9d022071b9d0ca/gistfile1.txt)
- **Proton version:** 4.11-11

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.



## Symptoms 
Game works on gold
  - After 30 mins of game time i've experienced two hard freezes that required the game to be restarted
  - I was unable to select the the content in the tutorial scene in officer's textbook, restart of the game fixed this issues and i was unable to reproduce it after in three other attempts
- After finishing at the first location there is a side-quest with a women who drowned on the boat once this quest is finished and MC goes to the church to find the girl and return the side quest is still here which is unexpected and if MC talks to the character it gets stuck requiring a restart of the game to proceed further.
- Clues are not readable even on highest settings (May be a game issue)
![image](https://user-images.githubusercontent.com/11302521/72312354-29070700-3688-11ea-85ef-901616044854.png)

Maybe this might help https://sfx.thelazy.net/games/preset/8286/


## Reproduction
- Download the game and play for some time, expect the game to hard freeze
- Check the officer's text book in tutorial to see if you can select the text

**Comment by ipr** [comment]
Currently DX11 version runs fine for me with Proton 6.3 and 7.0, Mesa 21.3.5. 
Haven't had freezes like mentioned above.
Re-testing might be useful?

Edit: adding that further playing without issues on Proton 7.0, no crashes or freezes that were mentioned before.
